app/javascript/admin/controllers/creator_settings_controller.js

```diff
@@ -1,4 +1,5 @@
 import { Controller } from '@hotwired/stimulus';
+import { isLowContrast } from '../../utilities/color/contrastValidator';
```

Comment:
Refactor (blocking): Use the `@utilities` [webpack alias](https://webpack.js.org/configuration/resolve/#resolvealias).

e.g. 

https://github.com/forem/forem/blob/cbff6cb92f083788cb6bc9cfdb449cd51708d108/app/javascript/actionsPanel/actionsPanel.js#L2

Reply:
✅